custom_components/tahoma/config_flow.py

```diff
@@ -41,9 +52,15 @@ async def async_validate_input(self, user_input):
         username = user_input.get(CONF_USERNAME)
         password = user_input.get(CONF_PASSWORD)
 
-        async with TahomaClient(username, password) as client:
+        hub = user_input.get(CONF_HUB) or DEFAULT_HUB
+        endpoint = SUPPORTED_ENDPOINTS[hub]
+
+        async with TahomaClient(username, password, api_url=endpoint) as client:
```

Comment:
If we support multiple endpoints, shouldn't we rename this (and the whole integration???) as it's not restricted to TaHoma anymore?

Reply:
I have been thinking about rebranding to Overkiz (the platform used by all companies), however for the sake of discoverability I would prefer to keep it Somfy TaHoma.

We can add this context to the docs and I will work on auto discover so that it will be found more easily when added to core.